src/index.cts

```diff
@@ -110,3 +69,91 @@ export type Encrypted = {
   }
   v: number
 }
+
+export type EncryptConfig = {
+  v: number
+  tables: Record<string, Record<string, Column>>
+}
+
+export type Column = {
+  cast_as?: CastAs
```

Comment:
Noticed that quite a few of the fields in the encrypt config are still snake case. I'd like to refactor those to use camelCase. I'll handle that in a separate PR since this one is already getting large (and this change doesn't touch the encrypt config file, yet).